h2o-core/src/main/java/water/Iced.java

```diff
@@ -31,7 +31,7 @@
  * @see water.Weaver
  * @see water.AutoBuffer
  */
-abstract public class Iced<D extends Iced> implements Freezable<D>, Externalizable {
+abstract public class Iced<D extends Iced> implements Freezable<D> {
```

Comment:
Are you sure here? It needs more explanation - this can potentially  break lot of stuff